utils/logalloc.cc

```diff
@@ -1554,6 +1556,8 @@ class region_impl final : public basic_region_impl {
 
     void compact_segment_locked(segment* seg, segment_descriptor& desc) noexcept {
         auto seg_occupancy = desc.occupancy();
+        reclaim_timer timing_guard("compact_segment_locked", is_preemptible::no, seg_occupancy.used_space(), 1);
+
```

Comment:
Not sure about this. Yes, we see a stall for a specific segment compaction. But why is it interesting in which function the stall happened?

Reply:
https://github.com/scylladb/scylla/issues/6189#issuecomment-1114678869 ?

Reply:
We could report the reserve size, instead of making everything more complicated.

Reply:
...?

The added complication is not small, and it will have some small performance cost.

Reply:
Another way to look at it is that individual segments are anonymous, it doesn't help to know that a particular one took a long time.